feedthefox/users/templates/jinja2/account/account_inactive.html

```diff
@@ -5,5 +5,8 @@
 {% block content %}
   <h1>{{ _("Account Inactive") }}</h1>
 
-  <p>{{ _("This account is inactive.") }}</p>
+  <p>
```

Comment:
You could add `class="text-danger"` just highlight that this is an error message.